core/string/ustring.h

```diff
@@ -430,6 +431,7 @@ class String {
 	String c_escape_multiline() const;
 	String c_unescape() const;
 	String json_escape() const;
+	String bbcode_escape() const;
```

Comment:
```suggestion
	String escape_bbcode() const;
```

Reply:
Out of curiosity, why rename this method? We have `json_escape()` above already.

Reply:
You're right, I didn't notice that. Just verb + noun looks more standard. Given `c_escape()` and `json_escape()`, it is logical to name this method `bbcode_escape()`. However, should the second one then be named `bbcode_strip()` to be consistent with the first, or `strip_bbcode()` to be consistent with `strip_edges()` and `strip_escapes()`?

Reply:
I'd prefer keeping `strip_bbcode()` as well to be consistent with `strip_edges()` and `strip_escapes()`.